app/models/alchemy/picture.rb

```diff
@@ -88,6 +88,10 @@ def allowed_filetypes
       left_outer_joins(:taggings).where(gutentag_taggings: {id: nil})
     }
 
+    scope :not_used, -> {
+      left_outer_joins(:essence_pictures).where(essence_pictures: {id: nil})
```

Comment:
This does not work for me. The pictures with an assignment still appear in the filtered list.